style.css

```diff
@@ -148,6 +148,434 @@ body {
   cursor: pointer;
 }
 
+.middle-section {
+  min-height: 281px;
+  padding-bottom: 51px;
+  background: #FAE8E4;
```

Comment:
Codacy found an issue: [Expected "#FAE8E4" to be "#fae8e4" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)